app/assets/javascripts/transaction.js

```diff
@@ -89,9 +89,25 @@ ST.transaction = (function(_) {
     ).onValue(function () { window.location = redirectUrl; });
   }
 
+  function initializeFreeTransactionForm(locale) {
+    window.auto_resize_text_areas("text_area");
+    $('textarea').focus();
```

Comment:
"textarea" vs. "text_area"?
Could this function get the element as parameter instead of class search from the whole dom?